scripts/ci/pre_commit/pre_commit_check_deferrable_default.py

```diff
@@ -76,7 +76,7 @@ def iter_check_deferrable_default_errors(module_filename: str) -> Iterator[str]:
         args = node.args
         arguments = reversed([*args.args, *args.kwonlyargs])
         defaults = reversed([*args.defaults, *args.kw_defaults])
-        for argument, default in itertools.zip_longest(arguments, defaults, fillvalue=None):
+        for argument, default in zip(arguments, defaults):
```

Comment:
These are not equivalent.

Reply:
In the subsequent line:

```python
            if argument is None or default is None:
                continue
```

all the lines that come from `zip_longest` but are ignored in `zip` are ignored anyway.